src/flamenco/runtime/tests/fd_exec_instr_test.c

```diff
@@ -77,7 +77,7 @@ fd_exec_instr_test_runner_footprint( void ) {
   ulong l = FD_LAYOUT_INIT;
   l = FD_LAYOUT_APPEND( l, alignof(fd_exec_instr_test_runner_t), sizeof(fd_exec_instr_test_runner_t) );
   l = FD_LAYOUT_APPEND( l, fd_funk_align(),                      fd_funk_footprint()                 );
-  return l;
+  return FD_LAYOUT_FINI( l, alignof(fd_exec_instr_test_runner_t) );
```

Comment:
should be `fd_exec_instr_test_runner_align()`

Reply:
I agree, we should be more careful across the codebase in using the provided functions for easier maintainability. I changed that + I made extra changes where this is happening elsewhere, specifically where the LAYOUT does not match the alloc, to make it clearer to read and eventually to statically check on a syntax level